StudentManagement/StudentManagement.Console/Program.cs

```diff
@@ -1,18 +1,41 @@
-﻿using StudentManagement.Console;
+﻿using Newtonsoft.Json;
+using StudentManagement.Console;
+using System.Text;
+
+
 
 bool menu = true;
 var students = new List<Student>();
 string name, surname, occupation;
 int age, id;
 
+
+
 do
 {
+
+    string filePath = @"C:\Users\macie\source\repos\StudentManagement\file2.json";
```

Comment:
1. The file should be loaded outside do while loop - why you want to load it every time when you start the app? 
2. This file won't exist on my PC ever and will throw errors.

Reply:
i changed that and now a filepath is located in my program before do-while loop